### DISCUSSION TITLE
Transmission VPN stuck on bad port, nothing downloads

### DISCUSSION THREAD

**Discussion by thegameksk** [haugene/docker-transmission-openvpn#2805] [Q&A]
Everything was working fine until a few days ago. I noticed nothing is downloading. Log shows the container says error: couldn't test port: bad request.

[_transvpn-transmission-openvpn-1_logs.txt](https://github.com/haugene/docker-transmission-openvpn/files/14250944/_transvpn-transmission-openvpn-1_logs.txt)

version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - /srv/mergerfs/nas/downloads:/data
            - /srv/mergerfs/nas/appdata/transmission-vpn:/config
        environment:
            - OPENVPN_PROVIDER=PIA
            - OPENVPN_CONFIG=USA
            - OPENVPN_USERNAME=*************
            - OPENVPN_PASSWORD=************
            - LOCAL_NETWORK=192.168.1.0/24
            - OPENVPN_OPTS=OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn

**Comment by pkishino** [comment]
The port script for Pia looks like it can’t get a response/connection refused. Needs to be updated and would be good if better error handling is added. For now you can either wait and see if it resolves (could be Pia is doing maintenance?) or disable the script and manually set the forwarding port

**Reply by thegameksk** [reply]
How would I disable script and manually set the forwarding port?

**Comment by pkishino** [comment]
Read the documentation on config options, the port can be set via the gui
among others

**Reply by thegameksk** [reply]
I foud this poking around https://github.com/haugene/docker-transmission-openvpn/issues/1507#issuecomment-726397923. The test torrent that is posted there works fine but nothing else downloads.
